protoc-gen-hack/plugin.go

```diff
@@ -232,10 +228,7 @@ func writeBinaryFileDescriptor(fdp *desc.FileDescriptorProto, resp *ppb.CodeGene
 		panic(err)
 	}
 
-	// The protoc compiler will complain that this contains invalid UTF-8 data,
-	// but we will ignore that for now.
```

Comment:
This is the reason we need this PR: the complain is turning into an error in newer version of the `protoc`.